documentation/docs/overview.md

```diff
@@ -7,29 +7,62 @@ keywords:
 - low level
 - stronghold
 - value transactions
+- explanation
 ---
 # Overview
 
-To communicate with the IOTA network, you have to connect and interact with a [node](https://wiki.iota.org/chrysalis-docs/node_software), the task that the iota.rs library is designed to simplify. It will generate addresses for you, send messages, sign and send transactions, and more.
+The iota.rs library is designed to simplify how you connect and interact
```

Comment:
Just for clarity sake: this phrasing is perfectly fine because it says that the task will be simplier than without iota.rs library, but not necessary simple.